src/BaseTemplates/EmptyWeb/Program.cs

```diff
@@ -15,6 +15,7 @@ public static void Main(string[] args)
             var hostingConfiguration = WebApplicationConfiguration.GetDefault(args);
 
             var application = new WebApplicationBuilder()
+                .UseServerFactory("Microsoft.AspNet.Server.Kestrel")
```

Comment:
The recommended way is to have this in code instead of using the hosting.json file. I can revert this if you prefer to keep hosting.json